LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -234,6 +236,14 @@ static function GetUpdatedHitChances(X2AbilityToHitCalc_StandardAim ToHitCalc, o
 	}
 }
 
+static function UpdateTeamwork(X2AbilityTemplate Template)
+{
+	Template.AbilityCosts[0].bConsumeAllPoints = true;
+	Template.AbilityTargetConditions.AddItem(default.GameplayVisibilityCondition);
```

Comment:
This also doesn't work. `GameplayVisibilityCondition` is only available to subclasses of `X2Ability`.